javascript/javascript1/week1/smart-ease.io.js

```diff
@@ -0,0 +1,104 @@
+const myFreeCodeCampUserName = "https://www.freecodecamp.org/sudar";
+
+const yearOfBirth = Number(prompt("enter your year of birth"));
+const yearFuture = Number(prompt("enter the year in a future"));
+const age = yearFuture - yearOfBirth;
+alert(`You will be ${age} years old in ${yearFuture}`);
+
+const dogYearOfBirth = Number(prompt("enter your dog's year of birth"));
+const dogYearFuture = Number(prompt("enter the year in a future"));
+const shouldShowResultInDogYears = Boolean(
+  prompt(
+    "Enter 0 if you want ot calculate dog's age in human years. Enter any other value if you want it in dog-years:"
+  )
+);
+let dogAge = dogYearFuture - dogYearOfBirth;
+let dogYear = "human";
+if (shouldShowResultInDogYears) {
+  dogAge *= 7;
+  dogYear = "dog";
```

Comment:
I don't understand why you declared this variable `dogYear`.

Reply:
It's here to put the right message later - Just a place holder for the relevant part of text.